src/sas/sascalc/calculator/ausaxs/sasview_sans_debye.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import logging
+
+def sasview_sans_debye(q, coords, weight, worksize=100000):
+    """
+    Compute I(q) for a set of points using the full Debye formula.
+    *q* is the q values for the calculation.
+    *coords* are the sample points.
+    *weight* is the weight associated with each point.
```

Comment:
Since you are describing all arguments, please also add description for `worksize`